### ISSUE TITLE
TidySheet5e Improper Labeling with Abilities

### ISSUE THREAD

**Issue by DarkSerpent** [misthero/dnd5e-custom-skills#10]
**Foundry Version:** 9.225
**D&D5e System Edition:** 1.6.3

Hello! I'm loving the concept of the module, but I'm finding difficulty when it comes to adding new abilities to TidySheet5e, which should have full support. The following label appears after applying changes to world and reloading:

![image](https://user-images.githubusercontent.com/12854153/188024262-5fe91980-54c4-4170-9849-78599715d791.png)
![image](https://user-images.githubusercontent.com/12854153/188024460-765f21d9-516c-4551-b229-1cfd766982a9.png)

This problem _only_ affects TidySheet5e, as the Default Sheet is working as intended:

![image](https://user-images.githubusercontent.com/12854153/188024311-b96c0c2c-b386-4baa-be88-0f83c1630a47.png)

Thank you for your continued development on the module!

**EDIT:** I believe this has to do with the fact that TidySheet5e shortens the names of abilities, unlike the Default Sheet. For instance: "Intelligence" becomes "INT".

**Comment by misthero** [comment]
thankyou, it was probably a regression in latest release, can you try to update to v1.0.8?

**Comment by DarkSerpent** [comment]
Hello! I attempted to update to v1.0.8, but Foundry reports an error:

![image](https://user-images.githubusercontent.com/12854153/188337438-833bd9fc-f628-4e3e-8fd9-f8c18c4efa6e.png)

So instead, I installed the module manually, and I can confirm the update does indeed work!

![image](https://user-images.githubusercontent.com/12854153/188337536-a864288a-2d12-45d7-83a1-e4afe97f9609.png)

Thanks for the quick fix!
